README.md

```diff
@@ -45,6 +43,19 @@ See [`Cargo.toml`](./Cargo.toml) for the configuration passed to [docs.rs] for b
 [`rustdoc` documentation]: https://doc.rust-lang.org/rustdoc/command-line-arguments.html
 [Bevy game engine]: https://bevyengine.org
 
+## Trait Tags
+
+This extension adds tags on types to indicate usage based on Bevy traits such as `Component`, `Event`, etc.
+
+For trait tags to show in listings, metadata needs to be embedded within the source files. You likely do not want to commit this to version control on a branch that receives new manual changes.
+Run the `embed-trait-info` tool and point it at your project workspace, e.g.
+
+```bash
+embed-trait-info my_crate
+```
+
+before building the documentation as above.
```

Comment:
I might be using this wrong, but I needed to specify the root directory with `--root-dir`, and also pass in the names of the packages to modify. Ultimately I got it working with this:
```bash
cargo run --bin main -- --root-dir ../ bevy_docs_extension_demo
```
I haven't worked with binaries in Rust much, so this could probably be simplified, but maybe the expected arguments could still be described a bit more here?